src/models/transfers.js

```diff
@@ -481,11 +473,27 @@ function * insertTransfers (externalTransfers) {
     converters.convertToInternalTransfer))
 }
 
+function * resendNotification (transferId) {
```

Comment:
Good points, but actually I don't think it's a problem because:
* When a request is send, a lookup is made in the database. If the transfer exists, the notification will first be emitted using the event name `transfer-{$ACCOUNT}`. See https://github.com/interledger/five-bells-ledger/blob/ws_notifications/src/lib/notificationBroadcasterWebsocket.js#L52
* So the notification will only be sent to the ws connections that listen to that account, no worries there

That being said, if a client sends a request with a transfer id which correspond to a transfer that doesn't belong to that account:
* A notification will be sent to the clients listening for notifications on that account, even though they have not requested a new notification
* I don't think that is a problem as this is very unlikely to happen. The probability of collision for generated UUIDs is probably lower than the risk of being reincarnated as an olive in a next life

There is an issue with this implementation though. If 2 (or more) clients are listening for notifications for a given account and only one of the client requests a new notification for a given transfer. Then _all_ the clients will receive the notification, not only the requesting one. We could either:
* Change this behaviour (that would take some work)
* Add to the specs that notification may be sent more than once. Clients would therefore need to be idempotent